root/statistics/MusicBrainzHistory.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import expand2react from '../static/scripts/common/i18n/expand2react';
+
+type PropsT = {
+  +events: Array<StatisticsEventT>,
+};
+
+const MusicBrainzHistory = ({
+  events,
+}: PropsT): React.Element<typeof Layout> => {
+  const sortedEvents = events.reverse();
+
+  return (
+    <Layout fullWidth title={l('History')}>
+      <h1>{l('Our Glorious History')}</h1>
+      {sortedEvents ? sortedEvents
+        .map((event) => {
+          const title = exp.l(
+            '{date} - {title}',
+            {date: event.date, title: event.title},
+          );
+          return (
+            <div key={event.date}>
+              <h2>
+                {event.link ? (
+                  <a href={event.link}>{title}</a>
+                ) : title}
+              </h2>
+              <p>{expand2react(event.description)}</p>
+              <hr />
+            </div>
+          );
+        }) : (
+        l('It seems we have no history to show at all!')
```

Comment:
Would it make sense to leave this untranslatable, since it'll probably only get shown on dev servers?
```suggestion
        'It seems we have no history to show at all!'
```

Reply:
Maybe? I mean, I don't know if we want our files to be usable by forks or whatnot, but if not this might be fine. Of course, I guess this would also be shown if we have an error loading the content, but in that case an English error might not be weird.